src/mscorlib/src/System/IO/File.cs

```diff
@@ -36,6 +36,11 @@ namespace System.IO {
     [ComVisible(true)]
     public static class File
     {
+        internal const int GENERIC_READ = unchecked((int)0x80000000);
+        private const int GENERIC_WRITE = unchecked((int)0x40000000);
+        private const int FILE_SHARE_WRITE = 0x00000002;
+        private const int FILE_SHARE_DELETE = 0x00000004;
+
```

Comment:
It is used in a number of classes. There are 22 direct live references- I'm going to assume that they expand from there.